app/src/test/java/org/simple/clinic/util/ErrorResolverTest.kt

```diff
@@ -103,32 +105,18 @@ class ErrorResolverTest {
   }
 
   @Test
-  @Parameters(method = "params for http unauthorized errors")
-  fun `http unauthorized errors must be identified correctly`(
-      httpException: HttpException,
-      expectedResolvedError: ResolvedError
-  ) {
-    val resolvedError = ErrorResolver.resolve(httpException)
-    assertThat(resolvedError::class).isSameAs(expectedResolvedError::class)
-    assertThat(resolvedError.actualCause).isSameAs(httpException)
-  }
+  fun `http unauthorized errors must be identified correctly`() {
+    // given
+    val exception = httpException(401)
```

Comment:
`401` - Unauthenticated?
`403` - Unauthorized?

Reply:
That is incorrect. If you look at https://httpstatuses.com/, the official terms are `Unauthorized` and `Forbidden`. I decided to follow that naming scheme.

Reply:
Discussed offline: Will rename `Unauthorized` -> `Unauthenticated`.